README.md

```diff
@@ -1,27 +1,163 @@
 # KubeVirt Enhancements Tracking and Backlog
 
-This repository is currently Work In Progress but will eventually be used to manage KubeVirt Enhancement Proposals (VEPs), emphasizing centralized prioritization and enhanced SIG involvement and collaboration.
+This repository is currently Work In Progress but will eventually be used to manage KubeVirt Enhancement Proposals (
```

Comment:
I'd avoid mentioning that this is work-in-progress.
de-facto, old design proposals are deprecated. It's true that we don't officially commit to VEPs yet, but a new user should definitely pick it over the old design proposals.

Reply:
I agree